babel.config.js

```diff
@@ -1,5 +1,6 @@
 const allowedE2EModules = [
   'window',
+  'hyperformula*',
```

Comment:
Is `*` necessary after change to the hf npm module?

Reply:
I needed this to import languages here: https://github.com/handsontable/handsontable/pull/7823/files#diff-bae07df71dfecc0b0f016bb98971b965b624fc95c1e5e41ac8c9c0652ab18534R4